cocotb/triggers.py

```diff
@@ -161,11 +161,36 @@ def unprime(self):
 class Timer(GPITrigger):
     """Fires after the specified simulation time period has elapsed."""
     def __init__(self, time_ps, units=None):
+        """
+        .. Constructor. This RST comment works around sphinx-doc/sphinx#6885
+
+        Args:
+           time_ps: The time value. This can also be a 
+               :class:`fractions.Fraction` or a :class:`decimal.Decimal` instance.
+               Note that despite the name this is not actually in picoseconds
+               but depends on the *units* argument.
+           units (str, optional): One of
+               ``None``, ``'fs'``, ``'ps'``, ``'ns'``, ``'us'``, ``'ms'``, ``'sec'``.
+               When no *units* is given (``None``) the timestep is determined by
+               the simulator.
```

Comment:
I'm tempted to just refer to the docs for `get_sim_steps` here, rather than listing all the values.

Reply:
This would give us two indirections from the docs of the timeout feature, and would also need more cleanup because we already duplicate this info in other places. I'd say dupli/triplicate for now and not open another can of worms at this time.
The real fix would be ``SimTime`` from #1130 which unfortunately @ktbarrett has closed.

Reply:
Can we compromise and do both? So add
```
See Also:
    :func:`~cocotb.utils.get_sim_steps`
```

Reply:
Sure, done.